.docs/README.md

```diff
@@ -93,7 +93,7 @@ final class CreateUserHandler implements IHandler
         $user = new User($command->email);
         $this->em->persist($user);
 
-        return DataResult::of($user);
```

Comment:
I agree on everything, but `return DataResult::of($user)` vs `return new DataResult($user)`, I would like to keep `of` variant, or `from` it's ok for me.

It's compatible with your `return new DataResult($user)` and also with mine `return DataResult::from($user)`. Otherwise it's compatible only with your way.

Reply:
Okay, changed. I have also added test case for this method.